src/messages/data.rs

```diff
@@ -18,7 +17,7 @@ pub struct FieldDescription {
     // the attribute number of the column, default to 0 if not a column from table
     pub column_id: i16,
     // the object ID of the data type
-    pub type_id: Oid,
+    pub type_id: u32,
```

Comment:
```rust
pub type Oid = u32;
```
Oid is just an alias for u32, to avoid introducing `postgres_types`, we restore it here.